app/controllers/api/v2/subnets_controller.rb

```diff
@@ -43,6 +43,7 @@ def show
           param :from, String, :desc => N_("Starting IP Address for IP auto suggestion")
           param :to, String, :desc => N_("Ending IP Address for IP auto suggestion")
           param :vlanid, String, :desc => N_("VLAN ID for this subnet")
+          param :mtu, String, :desc => N_("MTU for this subnet")
```

Comment:
Should this perhaps be a `:number` instead of a string?

Reply:
As VLANID was a String, just copied it. Changed to Integer.